app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+              ${recentProjects[0].description}
+            </p>
+            <ul class="recent-buttons-container">
+              <li>
+                <a
+                  href="https://www.w3.org/Style/CSS/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-1"
+                  >CSS</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-2"
+                  >HTML</a
+                >
+              </li>
+              <li>
+                <a
+                  href="
+ https://getbootstrap.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-3"
+                  >Bootstrap</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-4"
+                  >Ruby</a
+                >
+              </li>
+            </ul>
+            <a
+              href="#0"
+              class="btn-see-project active-btn-effect hover-btn-effect pop"
+            >
+              See project
+            </a>`;
+  mostRecentProjectWrapper.appendChild(img);
+  mostRecentProjectWrapper.appendChild(div);
+  portfolio.appendChild(mostRecentProjectWrapper);
+  portfolio.appendChild(projectsWrapperDiv);
+  portfolio.appendChild(descktopProjectsWrapperDiv);
+
+  for (let i = 0; i < 6; i += 1) {
+    if (projectsWrapperDiv) {
+      projectDiv = document.createElement('div');
+
+      projectDiv.classList.add('project', 'project-block');
+      projectDiv.innerHTML = `  <h3 class="project-title">${recentProjects[1].name}</h3>
+            <p class="project-details">
+             ${recentProjects[1].description}
+            </p>
+            <ul class="project-technology">
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="project-btn project-btn-1 active-btn-effect hover-btn-effect"
+                >
+                  ${recentProjects[1].workStack[0]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://getbootstrap.com/"
+                  class="project-btn project-btn-2 active-btn-effect hover-btn-effect"
+                  > ${recentProjects[1].workStack[1]}</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="project-btn project-btn-3 active-btn-effect hover-btn-effect"
+                  > ${recentProjects[1].workStack[2]}</a
+                >
+              </li>
+            </ul>
```

Comment:
- [ ] 👏 As I mentioned earlier,   It's a valuable addition to showcase the technologies used in each project. However,  I noticed that you are currently limiting the display to only three technologies. In a real-life scenario, projects can have varying numbers of technologies, and it's essential to display them all. To improve this, I recommend making the technology list in the popup modal dynamic by using a loop as you did for dynamically generating the projects card. This way, you can display all the technologies for each project, regardless of their number. It will provide a more accurate representation of the technologies used in each project and enhance the user experience.

Keep up the good work and consider implementing this dynamic approach to showcase the technologies in the popup modal! 🚀💻🌟